plugins/modules/zammad_ticket.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+author:
+- Scaleup Technologies
+- Melvin Ziemann (@cloucs)
+
+module: zammad_ticket
+
+short_description: Create, update, close a Zammad ticket via the API
+
+version_added: "1.0.0"
+
+description: |
+  This module creates, updates, or closes a Zammad ticket using the API.
+  User credentials and ticket details are passed as parameters. This module can handle
+  ticket creation and updates based on the specified state. It also allows for the
+  closure of existing tickets.
+
+options:
+  fqdn:
+    description: The fully qualified domain name of the Zammad instance.
+    required: true
+    type: str
+  endpoint:
+    description: The API endpoint for the ticket management in Zammad.
+    required: true
+    type: str
+  api_user:
+    description: The username for authenticating with the Zammad API.
+    required: true
+    type: str
+  api_secret:
+    description: The password or API key for authenticating with the Zammad API.
+    required: true
+    type: str
+  state:
+    description: The desired state of the ticket. Use 'present' to create or update a ticket, and 'absent' to close a ticket.
+    required: true
+    type: str
+    choices: ["present", "absent"]
+  ticket_id:
+    description: The unique identifier of the ticket to update or close. Required if state is 'absent' or when updating a ticket.
+    required: false
+    type: str
+  customer:
+    description: The email address of the customer for the ticket.
+    required: true
```

Comment:
Würde das gerne so haben, dasss man auch nur einzelne Werte eines Tickets ändern kann ohne das man z.B. einen Body und ein Subject hinzufügen muss.

Also z.B. verschiebe das Ticket in eine andere Gruppe:
```
- name: Change group of Ticket 12345 to Finance
    zammad_ticket:
      fqdn: "https://zammad.example.com"
      endpoint: "/api/v1/tickets"
      api_user: "api_user"
      api_secret: "api_secret"
      state: "present"
      ticket_id: "12345"
      group: "Finance"
```

Das sollte dann auch nur changed_true geben, wenn die Gruppe vorher nicht schon so war.